cli/build/local.go

```diff
@@ -68,11 +78,32 @@ func buildLocal(cmdCtx *cmdcontext.CmdCtx, cliOpts *config.CliOpts, buildCtx *Bu
 	}
 
 	// Run rocks make.
-	rocksMakeCmd := []string{"make"}
+	log.Infof("Running rocks make")
+	logOutput := os.Stdout
+	if !cmdCtx.Cli.Verbose {
+		if logOutput, err = os.CreateTemp("", "app_build"); err != nil {
+			return err
+		}
+		defer logOutput.Close()
+	}
+
+	rocksMakeArgs := []string{"-c", cmdCtx.Cli.ConfigPath, "rocks", "make"}
 	if buildCtx.SpecFile != "" {
-		rocksMakeCmd = append(rocksMakeCmd, buildCtx.SpecFile)
+		rocksMakeArgs = append(rocksMakeArgs, buildCtx.SpecFile)
 	}
-	if err := rocks.Exec(cmdCtx, cliOpts, rocksMakeCmd); err != nil {
+	selfPath, err := os.Executable()
+	if err != nil {
+		return err
+	}
+
+	rocksMakeCmd := exec.Command(selfPath, rocksMakeArgs...)
+	rocksMakeCmd.Stdout = logOutput
+	rocksMakeCmd.Stderr = logOutput
```

Comment:
Looks like, there is an issues with current "hide build logs" `tt install` implementation and therefore `tt build`: it prints build logs to stdout, and will hide all errors from me, if I redirect stdout to dev null. But I must be able to see the error messages from stderr. And must be able to skip all non-error messages. Also, I think we must print only stderr output in case of error if `-V` is not set. If `-V` option is set - print everything even if the error is not encountered. So, let's avoid stderr redirection and do not print something from stdout if `-V` is not set (ioutil.Discard instead of a log file) . Please check that rocks make error are printed to stderr.
I'd like to do the same (not under this task) for `tt install`. @LeonidVas what do you think?

Reply:
Based on this, we can try to redirect or own stdout to null and do not run child tt process. So, luarocks make errors will be printed to stderr, stdout will be discarded. And if someone wants to see the full output, the `--verbose` option is available.

Reply:
> Looks like, there is an issues with current "hide build logs" `tt install` implementation and therefore `tt build`: it prints build logs to stdout, and will hide all errors from me, if I redirect stdout to dev null. But I must be able to see the error messages from stderr. And must be able to skip all non-error messages. Also, I think we must print only stderr output in case of error if `-V` is not set. If `-V` option is set - print everything even if the error is not encountered. So, let's avoid stderr redirection and do not print something from stdout if `-V` is not set (ioutil.Discard instead of a log file) . Please check that rocks make error are printed to stderr. I'd like to do the same (not under this task) for `tt install`. @LeonidVas what do you think?

This sounds good to me.

Reply:
Implemented in this way.